### PR TITLE
Fix certificate yaml for multiple additionalDomains

### PR DESCRIPTION
**Pull request by stefee**
Before: [link](https://helm-playground.com/#t=N7C0AICcEMDsHMCm4AkA6AatANgV0QM5rQAmJAlgC7kD2sOAIjQLbTmwHigC%2B3AUNAAO5DIkgFasAFzgAxmMqhW9JJDS0A9ADcAjHwDW7EjIDCC8gDNys6JUR8CgxLKl9wclszoA5aM0QyIOBo4Lxu4CQcvv4Eru7uEABEAFRoQSG8ieEARogWNJCIAKKwxuCJdgSUWXwgEIilXGF8AhZ2kCVlFYTVfEA&v=LQhQEMBNISwFxgewHbgDYBFEFtw2QM4BcoABMKXAKYFwB0AxjmRdbQEyPPmU1wDMXbEA)
After: [link](https://helm-playground.com/#t=N7C0AICcEMDsHMCm4AkA6AatANgV0QM5rQAmJAlgC7kD2sOAIjQLbTmwHigC%2B3AUNAAO5DIkgFasAFzgAxmMqhW9JJDS0A9ADcAjHwDW7EjIDCC8gDNys6JUR8CgxLKl9wclszoA5aM0QyIOBo4Lxu4CQcvv4Eru7uEABEAFRoQSG8ieEARogWNJCIAKKwxuCJdgSUWXxBiKVcYXwCFnaQJWUVhNV8QA&v=LQhQEMBNISwFxgewHbgDYBFEFtw2QM4BcoABMKXAKYFwB0AxjmRdbQEyPPmU1wDMXbEA)

### CODE REVIEW

File: charts/bandstand-web-service/templates/certificate.yaml [hunk 1]
```diff
@@ -24,4 +24,4 @@ spec:
   privateKey:
     algorithm: "RSA"
     size: 2048
-{{- end -}}
+{{ end -}}
```
Comment (gavinclarkeuk) [inline charts/bandstand-web-service/templates/certificate.yaml:27]: Pretty sure you'll need a separator too
```suggestion
---
{{ end -}}
```

**Comment by gavinclarkeuk** [comment]
Can you also add a test case under the test-charts directory. Will need adding to the build. Then you can review the output

**Comment by stefee** [comment]
Thanks - added a test case now

**Comment by gavinclarkeuk** [comment]
I don't think this will work, as all the certificates all have the same resource name - need to add a suffix. Something like [this](https://mustafaak4096.medium.com/loops-in-helm-charts-259e1b9e8422#:~:text=Range%20function%20is%20used%20to,index%20of%20the%20current%20iteration.)

**Comment by stefee** [comment]
Thanks @gavinclarkeuk !

Latest: [link](https://helm-playground.com/#t=LQhQG92ACAnBDAdgcwKbQCQEsA0mBu8ANgK7oBcAvNAHQBqxZAzjfACZtYAuWA9osQAivALbwsiJtGABfGaHgAHLHVSwmfROWgBjNV2BiBaWDT4B6fAEZQAawlttAYX1YAZlh3wuqUCNRc7N7w5KDQ0AL%2B2nqwBpCYWNByoEyKqDqh4TqiIvwAcvBR0PEYhKToyeFskgX%2BTJnh0tAARABUNCVlZEkyzWHQAEaobrywqACiiI4tPkxcfSDAEODQqFPSyaAKbj6wk9PNs-OgQA&v=LQhQEMBNISwFxgewHbgDYBFEFtw2QM4BcoABMKXAKYFwB0AxjmRdbQEyPPmU1wDMXbEA)

**Pull request closed without merging**